service/src/main/kotlin/xyz/lebkuchenfm/domain/xsounds/XSoundsService.kt

```diff
@@ -19,15 +19,30 @@ class XSoundsService(private val repository: XSoundsRepository, private val file
     }
 
     sealed interface NewSoundError {
-        data object CouldNotAddNewSound : NewSoundError
+        data object FileStorageError : NewSoundError
+        data object DataBaseError : NewSoundError
```

Comment:
Database is a single word 😉 
```suggestion
        data object DatabaseError : NewSoundError
```